routes/xref/update.js

```diff
@@ -0,0 +1,81 @@
+const bodyParser = require("body-parser");
+const crypto = require("crypto");
+const { exec } = require("child_process");
+
+const bikeshedSecret = process.env.BIKESHED_SECRET;
+if (!bikeshedSecret) {
+  throw new ReferenceError("env variable `BIKESHED_SECRET` is not set.")

```

Comment:
I think just `Error` is fine here... otherwise it can be confusing distinguishing between JS engine errors and our errors.  

```suggestion
  throw new Error("env variable `BIKESHED_SECRET` is not set.")
```